.github/workflows/early_final_release_checker.yml

```diff
@@ -107,14 +107,12 @@ jobs:
     needs:
       - archive-run-small
       - archive-notify
-    env:
-      OUTPUT1: ${{needs.archive-notify.outputs.output1}}
-      # If Slack message includes new data, create a Github issue
-    if: ${{ always() && (contains(OUTPUT1, "CREATE") || contains(OUTPUT1, "UPDATE") || inputs.create_github_issue == true) }}
     steps:
       - uses: actions/checkout@v4
       - name: Create an issue
         uses: JasonEtco/create-an-issue@v2.9.2
```

Comment:
Not blocking at all but I wonder if using the `gh` CLI for things like creating issues (or any other GitHub API interaction) would be a cleaner and more flexible solution?

https://docs.github.com/en/actions/using-workflows/using-github-cli-in-workflows

Reply:
Quite possibly, but no idea how this would interact with the environment variables I'm passing and I've already probably spent too much time on this, so I'll shelf this proposal for now.